File: data/min_pushes/LeetCode_Solutions/Original_Solution.java
class Solution {
  public int minimumPushes(String word) {
    int[] count = new int[26];
    for (char c : word.toCharArray()) {
      count[c - 'a']++;
    }

    int push = 0;
    Arrays.sort(count);
    for (int i = 0; i < 26; i++) {
      push += count[25 - i] * (i / 8 + 1);
    }

    return push;
  }
}

File: data/min_pushes/LeetCode_Solutions/Original_Solution.py
class Solution:
    def minimumPushes(self, word: str) -> int:
        count = 1 
        d = {}
        g = Counter(word)
        count1 = 0
        l = sorted(g.items(),key = lambda x:x[1])
        l = l[::-1]
        for i in range(len(l)):
            if count<9:
                d[l[i][0]] = 1
                count+=1
            elif count<17:
                d[l[i][0]] = 2
                count+=1
            elif count <25:
                d[l[i][0]]=3
                count+=1
            else:
                d[l[i][0]] = 4
                count+=1
        for i in word:
            count1+= d[i]
        return count1
            